schema.graphql

```diff
@@ -59,6 +59,19 @@ type AuthData {
   user: User!
 }
 
+type CampaignPledge {
+  _id: ID!
+  amount: Float!
+  campaigns: [FundCampaign]
```

Comment:
You have not created the type definitions properly, which is leading to the items being nullable and thus, the type errors you are getting. Refer to the comments.